plugins/modules/keycloak_authentication.py

```diff
@@ -24,6 +24,8 @@
         support: full
     diff_mode:
         support: full
+    action_group:
+        version_added: 11.0.0
```

Comment:
Ooops, I think I put the wrong version number in my post :-( It should be 10.2.0, not 11.0.0, since this can go into the next minor release (instead of the next major release) already.
```suggestion
        version_added: 10.2.0
```
Same for the other modules.